src/goMain.ts

```diff
@@ -57,8 +57,16 @@ export function activate(ctx: vscode.ExtensionContext): void {
 
 	updateGoPathGoRootFromConfig().then(() => {
 		const updateToolsCmdText = 'Update tools';
-		const prevGoroot = ctx.globalState.get('goroot');
-		const currentGoroot = process.env['GOROOT'];
+		interface GoInfo {
+			goroot: string;
+			version: string;
+		}
+		const toolsGopath = getToolsGopath() || getCurrentGoPath();
```

Comment:
In Go versions less than 1.8, it is possible that gopath will not be set, so `toolsGopath` here would be null or undefined

Reply:
@ramya-rao-a is this still a concern given that 1.12 is out and we dont test for 1.8 anymore

Reply:
Only thing that comes to mind is the future when modules have taken over and `GOPATH` becomes obsolete.  It's a larger issue though and I don't think it needs to hold up this PR.